tests/automation/setup/create_group.ts

```diff
@@ -21,8 +22,10 @@ export const createGroup = async (
   windowC: Page,
 ): Promise<Group> => {
   const group: Group = { userName, userOne, userTwo, userThree };
-  const emptyStateGroupText = englishStrippedStr('groupNoMessages')
-    .withArgs({ group_name: group.userName })
+  const groupUpdateInviteControlText = englishStrippedStr(
+    'groupInviteYouAndMoreNew',
```

Comment:
This should be `groupMemberNewTwo`

Not including the admin in the control message on groups initial creation anymore